### PR TITLE
Feature/back button my account

### PR DESCRIPTION
**Pull request by arturmagalhaesjr**

### CODE REVIEW

**Review by polishq — CHANGES_REQUESTED** [review]
A few requests:

- Please remove any commented code lines and console.log statements.
- I'm seeing a few lint warnings/errors in various files. They're unrelated to your changes but they should be cleaned up regardless. For example in `react/ProductSummaryWishlist.tsx` line 120 there's a "prefer early return" warning. 
- Please update the `vtex.render-runtime` manifest dependency to 8.x
- Your linter should be auto-formatting the changelog (headers should have line breaks before and after)
- Please rename `react/typings/vtex.my-account-commons.d copy.ts` to `react/typings/vtex.my-account-commons.d.ts`

**Review by arturmagalhaesjr — COMMENTED** [review]
done

File: react/AddProductBtn.tsx [hunk 1]
```diff
@@ -340,19 +341,25 @@ const AddBtn: FC<AddBtnProps> = ({ toastURL = '/account/#wishlist' }) => {
...
-    data?.checkList?.inList === false &&
-    wishListed.length !== 0 &&
-    wishListed.indexOf(productId) !== -1
-  ) {
-    const indexWishListed = wishListed.indexOf(productId)
-    wishListed.splice(indexWishListed, 1)
-    // localStore.setItem('wishlist_wishlisted', JSON.stringify(wishListed))
-    saveToLocalStorageItem(wishListed)
   }
+  // } else if (
```
Comment (polishq) [inline react/AddProductBtn.tsx:353]: Please remove this commented code

**Review by polishq — CHANGES_REQUESTED** [review]

File: react/AddProductBtn.tsx (hunk 1 above)
Comment (polishq) [reply react/AddProductBtn.tsx:353]: (and the following commented lines)

**Review by arturmagalhaesjr — COMMENTED** [review]
done

**Review by polishq — APPROVED** [review]

**Pull request merged**